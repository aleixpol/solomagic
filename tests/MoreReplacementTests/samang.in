\ref something0
\mt somethingsomething saman somethingsomething

\ref something1
\mt somethingsomething samantha somethingsomething

