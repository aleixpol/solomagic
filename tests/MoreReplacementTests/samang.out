\ref something0
\mt somethingsomething saman somethingsomething
\ma somethingsomething samang somethingsomething

\ref something1
\mt somethingsomething samantha somethingsomething
\ma somethingsomething samantha somethingsomething
